app/managers.py

```diff
@@ -0,0 +1,43 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema.db3")
+        self.table_name = "actors"
+
+    def all(self) -> list:
+        actors_cursor = self._connection.execute(
+            f"SELECT id, first_name, last_name FROM {self.table_name}"
+        )
+        return [Actor(*row) for row in actors_cursor]
+
+    def create(self, first_name_: str, last_name_: str) -> None:

```

Comment:
why didn't you write `first_name`? You don't need these extra underscore, same with last name